README.md

```diff
@@ -21,3 +21,10 @@ jekyll serve
 ```
 
 View the local site at http://localhost:4000.
+
+### Resizing icons
+
+To resize all the PWA icons in images/:
+1. Install [imagemagick](https://www.imagemagick.org/script/download.php)
```

Comment:
Can you do this with `brew` or does it require an executable?

Reply:
(Also, I'm surprised that there isn't an `npm` module that resizes images w/o a separately-installed dependency?)

Reply:
Yeah, `brew install imagemagick`. I didn't include it in the instructions cause I didn't want to be OS-specific 🍎

Reply:
Yeah, there probably is. I just used imagemagick because it is the swiss army knife for image stuff. I'll see if I can find a replacement for this